packages/dom/src/document.rs

```diff
@@ -67,6 +67,7 @@ pub trait DocumentLike: AsRef<Document> + AsMut<Document> + Into<Document> + 'st
     }
 }
 
+// TODO (Matt): change document IDs to be u64 for accesskit compatibility.
```

Comment:
I suspect we won't want to do this. `usize` is convenient for indexing. Probably at some point we'll want our own `NodeId` newtype. Then we can implement indexing and from/into conversions for it, and gain some type safety in the process.